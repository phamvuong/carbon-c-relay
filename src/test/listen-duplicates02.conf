
listen
	linemode
		[::]:2013 proto tcp
		0.0.0.0:2013 proto tcp
		[::1]:2013 proto tcp
		127.0.0.1:2013 proto tcp
	;
