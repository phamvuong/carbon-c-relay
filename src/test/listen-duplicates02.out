listen-duplicates02.conf:8:1: duplicate listener ::1 with :: for port 2013
	;
	^
failed to read configuration 'listen-duplicates02.conf'
